src/v/cluster/health_monitor_frontend.h

```diff
@@ -71,6 +76,10 @@ class health_monitor_frontend {
     ss::future<cluster_health_overview>
       get_cluster_health_overview(model::timeout_clock::time_point);
 
+    ss::future<std::error_code> maybe_refresh_cluster_health();
```

Comment:
This API is leaking internals of health monitor to the caller. Why would caller trigger health update. There is already an API where caller may request health monitor to force refresh its caches.

Reply:
You are right though, I don't love the layering here.

Reply:
Next version will eliminate exposing this. Slightly less efficient but better layering and less fragile APIs.

Reply:
Was this addressed?

Reply:
Resolved == addressed.

Reply:
Yes, that is why I resolved it. As requested I'll let the OP close the thread, thank you.